server/queries.js

```diff
@@ -0,0 +1,30 @@
+const { Client } = require('pg');
```

Comment:
might want a `use strict;` at the top of this file - we're using it in `index.js` for code quality so might as well here too